<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="houzf-PC" name="com.test.Timeout" tests="1" failures="0" timestamp="31 五月 2018 15:17:48 GMT" time="2.024" errors="1">
  <testcase name="test1" time="2.024" classname="com.test.Timeout">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.test.Timeout.test1() didn&#039;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.test.Timeout.test1() didn't finish within the time-out 2000
]]>
    </error>
  </testcase> <!-- test1 -->
</testsuite> <!-- com.test.Timeout -->
